README.md

```diff
@@ -80,28 +77,32 @@ its selections directly on that array.
 
 The selections array will be initialized to an empty array if not present.
 
-## Action and Action Arguments
+## Actions and Action Arguments
 
-The action that is dispatched by x-select whenever the selected value or values
-change (change event) has a function signature of:
+All of x-selects actions are closure actions. This means you must use
+the `action` helper (i.e. `on-click=(action "onClick")`). The function
+that is dispatched by x-select whenever the event fires has a function
+signature of:
 
 ```js
 /**
-* @param value {Object} the value selected by the user.
-* @param component {Ember.Component} the x-select component itself
+* @param {Object} value - the value selected by the user.
+* @param {Object} event - the jQuery event of the action
+* @param {Object} componentState - object that contains `isDisabled`,
+  `isMultiple`, `isRequired` & `isAutoFocus`
 */
-function (value, component) {
+function (value, event, componentState) {
   // action body...
 }
 ```
 
 Most of the time all you need is the value that has been selected, but
 sometimes your action requires more context than just that. In those
-cases, you can associate arbitrary attributes with the component
-itself and use them later inside your action handler.  For example:
+cases, you can use the `componentState` argument. It contains a couple
```

Comment:
I'm a little confused about `componentState` thing. Should there be an actual argument to the component called `componentState`? In the example, I just see `action` and `required`